lib/ecto/multi.ex

```diff
@@ -99,6 +99,7 @@ defmodule Ecto.Multi do
   defstruct operations: [], names: MapSet.new
 
   @type run :: (t -> {:ok | :error, any}) | {module, atom, [any]}
+  @type run_changeset :: (Map.t -> Changeset.t | Ecto.Schema.t)
```

Comment:
`Map.t` should be `map`. Maybe we could give this type a better name? `changeset_fun`? I don't think we should call it run because even the return type is different.